lib/components/molecules/ToolTip/Tooltip.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+/**
+ *
+ * ToolTip
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Tooltip.style';
+import type { Props, Div } from './types';
+
+const Tooltip = ({ tootTipContent, domElement, showImage, ...otherProps }: Props): Div => {
+  return (
+    <div
```

Comment:
Where is the trigger element for this tooltip? and also the close button?

Reply:
Have implemented hover and it is in style file.